spec/requests/api/v1/sessions/create_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe "POST api/v1/users/sign_in", type: :request do
+  before(:each) do
+    @user = FactoryBot.create(:user) 
+    @sign_in_url = '/api/v1/users/sign_in' 
+    @sign_in_params = {
+      user: {
+        email: @user.email,
+        password: @user.password
+      }
+    }
+  end
+
+  describe 'POST Create' do
+    context 'when login params is valid' do
+      before do
+        post @sign_in_url, params: @sign_in_params, as: :json
```

Comment:
Try extracting this into a `subject` variable above.
`subject(:signin) { post @sign_in_url, params: params, as: :json }`

Also, replace @sign_in_url with its corresponding coming from running this command: `rails routes`